game/marble/client/scripts/game.cs

```diff
@@ -939,7 +939,8 @@ function clientCmdSetGameState(%state, %data)
       // if we are in end state, write scores
       if (XBLiveIsStatsSessionActive() && (%state $= "end" || %state $= "wait"))
       {
-         if (%allowStats && $Client::currentGameCounts && %state $= "end")
+         //if (%allowStats && $Client::currentGameCounts && %state $= "end")
+         if (%state $= "end")
```

Comment:
I'm not 100% confident in this. We might want to figure out what those two parameters are and why they aren't being set properly. But this seems to work without causing any negative side effects (including singleplayer), so I did it for now.

Reply:
I would like to find out what those are for before merging this.

Reply:
I've reverted this change in favor of another change which enables the stats. This potentially has the downside of running extra xblive specific code which might print extra "command not found" things into the log, but everything seems to work fine including both singleplayer and multiplayer. I have no idea which change is better, so I'll leave that up to you. At least I think we know vaguely how those variables work, or at least why they weren't set before